README.adoc

```diff
@@ -585,6 +585,20 @@ $ just system-info
 This is an x86_64 machine
 ```
 
+- `just_executable()` - Path to the running just executable.
```

Comment:
`just_executable` isn't super similar to existing functions, so I think it should get it's own section after the `Justfile and Justfile Directory` section. Something like:

```
==== Just Executable

- `just_executable()` - Absolute to the just executable.
```